test/Microsoft.AspNet.Identity.EntityFramework.Test/DefaultPocoTest.cs

```diff
@@ -63,6 +65,152 @@ public void DropDatabaseStart()
             IdentityResultAssert.IsSuccess(await userManager.DeleteAsync(user));
         }
 
+        [Fact]
+        public async Task CanIncludeUserClaimsTest()
+        {
+            // Arrange
+            CreateContext(true);
+            var builder = new ApplicationBuilder(CallContextServiceLocator.Locator.ServiceProvider);
+
+            builder.UseServices(services =>
+            {
+                DbUtil.ConfigureDbServices<IdentityDbContext>(ConnectionString, services);
+                services.AddIdentity<IdentityUser, IdentityRole>().AddEntityFrameworkStores<IdentityDbContext>();
+            });
+
+            var userManager = builder.ApplicationServices.GetRequiredService<UserManager<IdentityUser>>();
+            var dbContext = builder.ApplicationServices.GetRequiredService<IdentityDbContext>();
```

Comment:
I am assuming you are worried how the database is cleared ? There is a test `DropDatabaseStart` that is marked with lowest priority that will delete the database after all the tests in the class have run. This class existed all the way long. I have only added more tests that's all. Regarding the object, since it is created in the scope of the test, then it should be disposed at the end of it. If adding a using statement for DBContext is the recommended practice then sure I can update that but will have to do it in other tests as well